app/workers/users/record_field_test_event_worker.rb

```diff
@@ -8,6 +8,7 @@ class RecordFieldTestEventWorker
     def perform(user_id, goal)
       @user = User.find_by(id: user_id)
       return unless @user
+      return if FieldTest.config["experiments"].nil?
```

Comment:
makes sense, but we _shouldn't_ normally get here (nothing would record a field test event) unless there were experiments?